containerized-accounts/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM openjdk:11-jdk-slim
 MAINTAINER Taner Diler <taner.diler@gmail.com>
-ADD target/containerized-accounts-1.0.0-SNAPSHOT.jar containerized-accounts.jar
-ENTRYPOINT ["java", "-jar", "/containerized-accounts.jar"]
-EXPOSE 2222
\ No newline at end of file
+ADD target/*.jar containerized-accounts.jar
```

Comment:
Bu çok doğru gelmedi bana. birden fazla jar varsa, en sonuncusu belki de bundle olmayan jar'ı içeri renaming yapıp alacak. bu yüzden pom.xml'de <build><finalName>containerized-accounts</finalName></build> tanımını yapmak daha mantıklı.

Reply:
Yeah, that's right. I just wanted to make it a little simpler. 👍